fastsim-core/src/vehicle/powertrain/efficiency_values.rs

```diff
@@ -0,0 +1,225 @@
+// values in the third sub-array (corresponding to power) in Altrios'
+// eta_interp_grid
+pub const X_EFF: &[f64] = &[-5.0, -3.0, -1.0, -0.5, -0.1, 0.1, 0.5, 1.0, 2.0, 3.0, 5.0];
+// values in the second sub-array (corresponding to SOC) in Altrios'
+// eta_interp_grid
+pub const Y_EFF: &[f64] = &[0.0, 0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9, 1.0];
+// values in the first sub-array (corresponding to temperature) in Altrios'
+// eta_interp_grid
+pub const Z_EFF: &[f64] = &[23.0, 30.0, 45.0, 55.0];
+// efficiency array as a function of power at constant 50% SOC and 23 degrees C
+// corresponds to eta_interp_values[0][5] in Altrios
+pub const F_X_EFF: &[f64] = &[
+    0.892085814015878,
+    0.935630351894604,
+    0.978612548997097,
+    0.989309767361546,
+    0.997862182940346,
+    0.997883658588665,
+    0.989234853052088,
+    0.977987897893482,
+    0.953873809588552,
+    0.927243266086586,
+    0.864025853742296,
+];
+
+// efficiency array as a function of power and SOC at constant 23 degrees C
+// corresponds to eta_interp_values[0] in Altrios, transposed so that the
+// outermost layer is now power, and the innermost layer SOC (in altrios,
+// the outermost layer is SOC and innermost is power)
+pub const F_XY_EFF: &[&[f64]] = &[
```

Comment:
@robinsteuteville 

For this large array, it'd be better to store it outside of rust with something like [include_str!](https://github.com/NREL/fastsim/blob/2f7e68971d088a4f56f6cca69dc7ca6646e317dd/fastsim-core/src/vehicle/vehicle_model.rs#L546) or [include_dir!](https://github.com/NREL/fastsim/blob/2f7e68971d088a4f56f6cca69dc7ca6646e317dd/fastsim-core/src/resources.rs#L2)

Reply:
@calbaker so to make sure I understand correctly, I should store the arrays as yaml files? I'm guessing I'll need to find something that deserializes the yaml file into a nested vector rather than a string?

Reply:
@robinsteuteville, ask Kyle what he thinks about this.  It could be the case that what I'm proposing is too onerous to be worth doing.

Reply:
@calbaker talked with Kyle, and he mentioned that the interpolator as a whole is serializable and deserializable and therefore each component also should be. However, I'm having a bit of trouble isolating the nested vector part to serialize and deserialize. One simple workaround could be to save in a yaml file an entire "default" interpolator for each dimension, rather than just saving separately each x, y, f_xy, etc. array. Then, each `set_default_xd_interp` method could access and deserialize one of the yaml files and set it to be the interpolator.

Reply:
> One simple workaround could be to save in a yaml file an entire "default" interpolator for each dimension,

Yes!  This!  @robinsteuteville